src/hotspot/cpu/x86/x86_32.ad

```diff
@@ -7827,7 +7827,7 @@ instruct divI_eReg(eAXRegI rax, eDXRegI rdx, eCXRegI div, eFlagsReg cr) %{
 // Divide Register Long
 instruct divL_eReg( eADXRegL dst, eRegL src1, eRegL src2, eFlagsReg cr, eCXRegI cx, eBXRegI bx ) %{
```

Comment:
I agree with @shipilev that we can remove `eFlagsReg cr, eCXRegI cx, eBXRegI bx` here, right?

Reply:
Done. I deleted them in the effect list but forgot to delete the corresponding args.